src/status_im2/common/parallax/whitelist.cljs

```diff
@@ -0,0 +1,12 @@
+(ns status-im2.common.parallax.whitelist
+  (:require [native-module.core :as native-module]
+            [clojure.string :as string]))
+
+(def device-id (:device-id (native-module/get-device-model-info)))
+
+(def whitelist ["iPhone11" "iPhone12" "iPhone13" "iPhone14"])
```

Comment:
I would make this a set,
if you do that, then you can simplify the below function

```
(def whitelist #{"iPhone11"....})

(defn whitelisted? []
   (let [device-type (first (string/split (str...))
      (whitelist device-type))
```

but I wonder why making it a runtime check?

```
(def device-id (:device-id...)
(def device-type (first (string/split (str device-id) ",")))
(def whitelist #{"iPhone"..})
(def whitelisted? (whitelist device-type))
```

Reply:
> but I wonder why making it a runtime check?

I agree with @cammellos. 


IMO, the best place would be `app-db`, because:
- we can add/update the `app-db` on db initialization (on app start) which can prevent the checks from running on every render 
- there will be a few more places (like other onboarding screens) where we will need this check (`whitelisted?`) for the parallax effect

Also, It's safe to have it on `app-db` as we don't need the user's permission for using the device's Gyroscope sensor (IOW, to have a whitelisted device check).

Reply:
yeah makes sense. I will add it to app-db as you suggest. Thanks!

Reply:
mmh, I am not sure, by `app-db` we mean the reframe db?
I would expect that to be slower than just having a boolean calculated only on startup that we check in render, as you have to build a subscription etc, which I'd assume is more logic to run

Reply:
@cammellos  Do you want a global atom stored on start up?  What file are you suggesting to include that in?

Reply:
@J-Son89 it would not be an atom, since it's not a mutable piece of data, it would just be a simple `def` (as per code above).

Where to store it, not sure, but we have similar examples if you look at `src/status-im/utils/platform`.

We use `platform/android?` in render methods always, and that's just a `(def` that is created at startup, this would be something similar, since it's not necessary to store in the app-db or an atom (generally, I would only use those if the value changes and we need to react to it, which is not the case here as it's not going to change).

`utils.platform?` would look like a place where we could have this, but also the file you just put it looks ok to me, I don't have strong opinions on the placement

Reply:
> mmh, I am not sure, by app-db we mean the reframe db?

Yes, the re-frame db.

I meant to have the check performed once on app startup, store the resultant **boolean** in app-db (re-frame db), and use it on other screens (which use this check for displaying parallax) via subscription. Sorry, I wasn't clear in the early comment/suggestion.

Yes, this might slightly increase the db initialisation time. Probably not a good suggestion. Apologies.

<hr />

Just a small addition to the @cammellos suggestion:

```clojure
(def ^:const device-id (:device-id (native-module/get-device-model-info)))

(def ^:const whitelist #{"iPhone11"....})

...
```

Add `^:const` to the `def` as it would reduce the [dereference time](https://stackoverflow.com/a/9165999) and increase the execution speed of the `whitelisted?` function.